dcat/index.html

```diff
@@ -58,16 +58,16 @@
         <li>more details for the ways of representing <a href="#examples-dataset-provenance">dataset provenance</a> and <a href="#quality-information">dataset quality</a></li>
         <li>an <a href="#dcat-sdo">alignment</a> between the DCAT vocabulary and the schema.org vocabulary</li>
     </ul>
-    
+
 
     <p>
         The detailed differences between the two documents can be seen <a href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20180508%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20181011%2F">here</a>
         and the list of all the changes since the previous version of DCAT in the <a href="#changes">Change History</a> section.
     </p>
 
     <h3 id="dcat_history">DCAT history</h3>
-    <p>The original DCAT vocabulary was developed and <a href="http://vocab.deri.ie/dcat">hosted</a> at the Digital Enterprise Research Institute (DERI), then refined by the <a href="http://www.w3.org/egov/">eGov Interest Group</a>, and finally standardized in 2014 [[VOCAB-DCAT-20140116]] by the <a href="http://www.w3.org/2011/gld/">Government Linked Data (GLD)</a> Working Group.</p>
-    <p>This revised version of DCAT was developed by the <a href="https://www.w3.org/2017/dxwg/">Dataset Exchange Working Group</a> in response to a new set of Use Cases and Requirements [[DCAT-UCR]] submitted on the basis of experience with the DCAT vocabulary from the time of the original version, and new applications not originally considered. A summary of the changes from  [[VOCAB-DCAT-20140116]] can be found at <a href="#changes">Change History</a></p>
+    <p>The original DCAT vocabulary was developed and <a href="http://vocab.deri.ie/dcat">hosted</a> at the Digital Enterprise Research Institute (DERI), then refined by the <a href="http://www.w3.org/egov/">eGov Interest Group</a>, and finally standardized in 2014 [[?VOCAB-DCAT-20140116]] by the <a href="http://www.w3.org/2011/gld/">Government Linked Data (GLD)</a> Working Group.</p>
```

Comment:
Should this one be normative?

Reply:
+1 from me

Reply:
So probably no ? or !, here, but add a mention in a normative section.

Reply:
Yes. There's however an issue with the current default behaviour of ReSpec: if a section is not explicitly marked with `class="informative"`, all the included references will be considered as normative, unless there's a ?. This is what happened in the UCR document, where all references ended up in the "normative references" section.

Reply:
OK - I understand the logic. Move the classification out to the sections (though not the document). You can downgrade from normative to informative, but can't upgrade. 

Could you put class="informative" on the \<body\> element for UCR?

Reply:
It appears that there is also a rule that references in a context with class="note" are informative by default, but can be promoted to normative with a !

Reply:
> Could you put class="informative" on the <body> element for a non rec-track document, like UCR?

I guess so.

Reply:
Reverted DCAT-2014 to non-normative ref